src/tasks/batch_integration/workflows/run_benchmark/main.nf

```diff
@@ -27,10 +27,23 @@ workflow run_wf {
     scanorama_feature,
     scanvi,
     scvi,
-    no_integration_batch,
-    random_embed_cell,
-    random_embed_cell_jitter,
-    random_integration
+    no_integration_batch_embed,
+    no_integration_batch_feature,
+    no_integration_batch_graph,
```

Comment:
These two control methods don't exist. Should they be removed from this list of dependencies, or do two more components need to be added?

Reply:
you're right, I missed that one